contracts/test/mocks/lib/token/TokenBalanceOfAllowanceMissingReturnMock.sol

```diff
@@ -0,0 +1,106 @@
+// Non-standards compliant token that is missing return values for
+// `allowance()` and `balanceOf()`.
+// Modified from https://github.com/OpenZeppelin/openzeppelin-solidity/blob/a9f910d34f0ab33a1ae5e714f69f9596a02b4d91/contracts/token/ERC20/StandardToken.sol
+
+pragma solidity 0.4.24;
+
+import "../../../../lib/math/SafeMath.sol";
+
+
+contract TokenBalanceOfAllowanceReturnMissingMock {
```

Comment:
WDYT about modeling this case within `TokenReturnMissingMock` as well?

Reply:
I was thinking about that... but what stopped me was in the `TokenReturnMissingMock` tests we need a working `balanceOf()` to check results against.